docker/verified-sources-fetcher/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM matterlabs/zksync-build-base:latest as builder
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+WORKDIR /usr/src/zksync
+COPY . .
+
+RUN cargo build --release --bin verified_sources_fetcher
+
+FROM debian:bookworm-slim
+RUN apt-get update && apt-get install -y apt-transport-https ca-certificates gnupg curl git && rm -rf /var/lib/apt/lists/*
```

Comment:
we need to run rm -rf /var/lib/apt/lists/* after google cli is installed to proper clean all cache